data/mods/innawood/npcs/missiondef.json

```diff
@@ -0,0 +1,421 @@
+    "type": "mission_definition",
+    "name": { "str": "Find Weather Log on Rock" },
+    "goal": "MGOAL_FIND_ANY_ITEM",
+    "difficulty": 2,
+    "value": 50000,
+    "item": "rock",
+    "assign_mission_target": { "om_terrain": "forest", "reveal_radius": 3, "search_range": 120 },
+    "update_mapgen": { "place_item": [ { "item": "rock", "x": 14, "y": 15, "amount": [ 10, 20 ], "repeat": [ 1, 3 ], "chance": 20 } ] },
+    "origins": [ "ORIGIN_OPENER_NPC", "ORIGIN_ANY_NPC" ],
+    "dialogue": {
+      "describe": "I wonder if a retreat might exist…",
+      "offer": "Everyone who dies gets back up, right?  Which means that whatever is causing this it must be airborne to have infected everyone.  I believe that if that is the case then there should be regions that were not downwind from where-ever the disease was released.  We need to find a rock with all the weather patterns leading up to the outbreak.",
+      "accepted": "Thanks so much, you may save both of us yet.",
+      "rejected": "Ya, it was a long shot I admit.",
+      "advice": "I'm not sure, maybe smashing up a big rock would give you the right rock?",
+      "inquire": "Any luck?",
+      "success": "These look more complicated than I thought, just give me some time.",
+      "success_lie": "This isn't what we need.",
+      "failure": "If only we could find a great valley or something."
+    }
+  },
+  {
+    "id": "MISSION_GET_RELIC",
+    "type": "mission_definition",
+    "name": { "str": "Find Rock" },
+    "goal": "MGOAL_FIND_ANY_ITEM",
+    "difficulty": 2,
+    "value": 100000,
+    "item": "rock",
+    "assign_mission_target": { "om_terrain": "forest", "reveal_radius": 3, "search_range": 120 },
+    "update_mapgen": { "place_item": [ { "item": "rock", "x": 14, "y": 15, "amount": [ 10, 20 ], "repeat": [ 1, 3 ], "chance": 20 } ] },
+    "origins": [ "ORIGIN_OPENER_NPC", "ORIGIN_ANY_NPC" ],
+    "dialogue": {
+      "describe": "St. Michael the archangel defend me in battle…",
+      "offer": "As the world seems to abandon the reality that we once knew, it becomes plausible that the old superstitions that were cast aside may have had some truth to them.  Please go and find me a religious relic… I doubt it will be of much use but I've got to hope in something.  It will look like a normal rock to your puny mind.",
+      "accepted": "I wish you the best of luck, may whatever god you please guide your path.",
+      "rejected": "Ya, I guess the stress may just be getting to me…",
+      "advice": "I hear rocks may be found in the woods.  They tell me things.  The spirits.  Religion!",
+      "inquire": "Any luck?  Please find me a small relic.  Any relic will do.  It looks like a rock.",
+      "success": "Thank you, I need some time alone now…",
+      "success_lie": "What good does this do us?",
+      "failure": "It was a lost cause anyways…"
+    }
+  },
+  {
+    "id": "MISSION_GET_SOFTWARE",
+    "type": "mission_definition",
+    "name": { "str": "Retrieve Stick" },
```

Comment:
A wooden spear or pike is also a stick & is something players might actually keep instead

Reply:
I can only define one item with the way missions are coded

Reply:
The Tacoma mission to get crutches for the doctor (a fairly late mission) works around the single item mission restriction by introducing a dialog option to ask if makeshift crutches are acceptable if you bring those. It might be possible to copy the logic used there (or it may be considered not worth the effort).

Reply:
That's a big lift for this PR but may be worth doing as a followup. Good to know, I've been wanting to do that for a DinoMod mission for a long time